trips.js

```diff
@@ -0,0 +1,111 @@
+
+const baseURL = 'https://trektravel.herokuapp.com/trips';
+
+
+$(document).ready( () => {
+
+  $('#book-trip-form').hide();
+
+  let loadTrips = function loadTrips() {
+    $.get(baseURL, (response) => {
+      response.forEach(function(trip)  {
+        if (trip.id < 35) {
+          let tripInfo = `<div class="column">
+          <h3 class="card" data-id=${trip.id}>
+          ${trip.name}</h3></div>`
+          $('#trips .row').append(tripInfo);
+        }
+      }); //for each
+    })
+    .fail(function(response) {
+      console.log(response);
+      $('#fail').html('<p>Your Request was Unsuccessful</p>')
+    })
+    .always(function(){
+      console.log("always and forever");
+    });
```

Comment:
While I appreciate the messages, you don't always need to add the `always` callback. I find it's only very occasionally useful, sort of like an `ensure` clause on a `begin/rescue` in Ruby.